misc/server/src/http_service.rs

```diff
@@ -38,21 +38,16 @@ pub(crate) async fn metrics_server(
     // Serve on localhost.
     let addr = ([0, 0, 0, 0], 8888).into();
 
-    // Use the tokio runtime to run the hyper server.
-    let rt = tokio::runtime::Runtime::new()?;
-    rt.block_on(async {
-        let server =
-            Server::bind(&addr).serve(MakeMetricService::new(registry, metrics_path.clone()));
-        info!(
-            "Metrics server on http://{}{}",
-            server.local_addr(),
-            metrics_path
-        );
-        if let Err(e) = server.await {
-            error!("server error: {}", e);
-        }
-        Ok(())
-    })
+    let server = Server::bind(&addr).serve(MakeMetricService::new(registry, metrics_path.clone()));
+    info!(
+        "Metrics server on http://{}{}",
+        server.local_addr(),
+        metrics_path
+    );
+    if let Err(e) = server.await {
+        error!("server error: {}", e);
+    }
```

Comment:
I think we can now just bubble this one up with `?`!

Reply:
But can we do it? Because `hyper::Error` is defined in external crate.

Reply:
Why does that matter? We should still be able to spawn it as a task onto the runtime?